exporters/otlp/otlpmetric/otlpmetrichttp/options.go

```diff
@@ -40,6 +40,14 @@ type Option interface {
 	applyHTTPOption(otlpconfig.Config) otlpconfig.Config
 }
 
+func asHTTPOptions(opts []Option) []otlpconfig.HTTPOption {
```

Comment:
(maybe for another PR too)
This is repeated quite a few times in both metric and tracer. How about sharing it?

Reply:
It is actually not a shared function. The arguments are slices of different types.